src/plugins/fieldflow/fieldflow.js

```diff
@@ -685,6 +685,11 @@ var componentName = "wb-fieldflow",
 				}
 			}
 		}
+		if ( isInline ) {
+			radCheckOut = "<div class='gc-form'>" + radCheckOut;
+		} else {
+			radCheckOut = "<ul class='list-unstyled gc-form'>" + radCheckOut;
```

Comment:
Rendering as a list items have some impact on the alignment of the first checkbox with the label. 

Working example with your code: https://universallabs.org/wet/labs/pr-1586/demos/fieldflow/basic-en.html
Downloadable zip file: https://universallabs.org/wet/labs/pr-1586.zip

<img width="723" alt="Screen Shot 2019-09-04 at 2 40 38 PM" src="https://user-images.githubusercontent.com/1676030/64281718-bf306100-cf21-11e9-850f-203d520c90c1.png">

Reply:
Whoops - definitely not as intended, will see about fixing it.